module/Finna/src/Finna/AjaxHandler/OnlinePaymentNotify.php

```diff
@@ -125,7 +128,11 @@ public function handleRequest(Params $params)
                 $this->ils->getMyProfile($patron)
             );
             try {
-                $this->receipt->sendEmail($user, $patronProfile, $t);
+                $res = $this->receipt->sendEmail($user, $patronProfile, $t);
+                $this->addTransactionEvent(
+                    $t->id,
+                    $res ? 'Receipt sent' : 'Receipt not sent (no email address)'
+                );
```

Comment:
Tässä ainakin heräs kysymys, et jos me logataan näitä virheitä kantaan, niin pitäiskö samalla myös tuol catchin sisällä laittaa exception mukana addTransactionEventtinä

Reply:
Joo, lisään.

Reply:
Nyt lisätty. Muita vastaavia puutteita en löytänyt.

Reply:
Joo, näytti olevan ainoa mikä puuttu